### DISCUSSION TITLE
极简首页功能有和我一样图片过大挤在一起情况的吗，之前版本这样设置是正常的，

### DISCUSSION THREAD

**Discussion by LINShen03** [the1812/Bilibili-Evolved#3675] [提问 / Q&A] [answered]
![image](https://user-images.githubusercontent.com/113405365/189796791-3a729f98-d209-43fc-8285-0d17b47f53cd.png)
![image](https://user-images.githubusercontent.com/113405365/189796985-b9acdf25-2494-49cd-be3f-dd08aa408460.png)
![image](https://user-images.githubusercontent.com/113405365/189797410-fdff9b7e-e055-4583-b6b3-52fd416f0604.png)
首先非常抱歉我是通过zip安装的，可能和这个有关。
但是原来正常情况（其他内容全部没有，只有动态呈两列显示，图片大小合适，标题内容文字可见）导出的设置在重新安装系统的电脑上导入后就如此。排除了几个插件都未能溯源，大致翻了一下讨论区好像没有类似的讨论，想寻求大家的帮助，谢谢。

**Comment by LINShen03** [comment]
系统分辨率、缩放已尝试均无关，笔记本、台式机同样问题
刷新有短暂显示正常，随后图片过大遮挡。
高分辨率图片已关闭

**Comment by LINShen03** [comment]
新旧版b站显示均同样问题，关闭其他所有插件仅保留极简首页，问题仍在。

**Comment by the1812** [comment, marked as answer]
你自己都说是 zip 装的了，还能是啥原因。。
![D6DDB493-AB96-4393-8F73-0FC39886653B](https://user-images.githubusercontent.com/26504152/189808463-6e5f6f3e-5bd4-4784-b2a2-b09b36964190.jpeg)

**Reply by LINShen03** [reply]
好的，谢谢作者大大，主要用的是您当时回复如何回退版本 #3655里面自动2.4.0更新回滚的2.2.4zip，以为还OK

**Reply by the1812** [reply]
这是因为 2.4.0 才加了极简首页，2.2.4 不能用

**Reply by LINShen03** [reply]
> 

/笑哭 好的好的，非常感谢
